src/app/cube/library/components/pagination/pagination.component.html

```diff
@@ -0,0 +1,24 @@
+<div class='library-items__pagination-container'>
+  <button 
+    *ngIf='!getPageNumberList().includes(1)'
+    class='library-items__pagination-container__arrow-button' 
+    (activate)='onLeftArrowClick()'
+    aria-label='to previous Saved Items page'>
```

Comment:
Since we want to make this component reusable the aria label should be an input that can be passed in. i.e. To previous {{ itemName }} where item name in the title passed into the component when it's put into markup.